pkg/commands/workload.go

```diff
@@ -229,6 +234,10 @@ func (opts *WorkloadOptions) ApplyOptionsToWorkload(ctx context.Context, workloa
 		workload.Spec.MergeSourceImage(opts.SourceImage)
 	}
 
+	if opts.SubPath != "" {
```

Comment:
it should be possible to unset an existing subpath.

Reply:
hmm very true.